nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedMetadataFacetSupport.java

```diff
@@ -106,6 +107,9 @@ public String generatePackageSnapshotAsJson(final ConanCoords coords) throws Jso
           return Stream.empty();
         })
         .collect(Collectors.toMap(Pair::getKey, Pair::getValue));
+    if (packageSnapshot.isEmpty()) {
+      return null;
```

Comment:
I would prefer to use `Optional.empty();` instead of `null`

Reply:
In other places the same situation. Using @Nullable